docker/parity_wrapper.sh

```diff
@@ -202,7 +202,13 @@ function runParity() {
   exec $PARITY_BIN $PARITY_ARGS
 }
 
+function copySpecFileToSharedVolume() {
+  echo "Copying trustline spec file to shared volume"
+  cp /config/trustlines-spec.json /config/chain-spec/trustlines-spec.json
```

Comment:
Why does it still copy to /config/chain-spec/ ?

Reply:
Apparently because I failed commiting because pre-commit complains that I do not have the executable `shfmt`